prime-router/metadata/hl7_mapping/receivers/STLTs/NY/NY-receiver-transform.yml

```diff
@@ -0,0 +1,66 @@
+hl7Class: ca.uhn.hl7v2.model.v251.message.ORU_R01
+
+extends: ../../../ORU_R01/ORU_R01-base
+elements:
+  # MSH 4-1
+  - name: sending-facility_namespace-id
+    condition: true
+    resource: 'Bundle'
+    value: [ '"CDC PRIME REPORTSTREAM"' ]
+
+  # MSH 4-2
+  - name: sending-facility-universal-id
+    resource: 'Bundle'
+    condition: true
+    value: [ '"10DRPTSTRM"' ]
+
+  # MSH-5
+  - name: receiving-application-namespace-id
+    condition: true
+    resource: 'Bundle'
+    value: [ '"ECLRS"' ]
+
+  # MSH-6
+  - name: receiving-facility-namespace-id
+    condition: true
+    resource: 'Bundle'
+    value: [ '"NYSDOH"' ]
+
+  # OBX CWE-CE
```

Comment:
Can we be a little more detail here to clarify that this is changing all OBX segments from CE to CWE data type? Also nitpicky, but can we move this below the MSH and SFT elements. Ideally, it is easier to read if the order of operations is in line with the order the elements appear in the output message.

Reply:
also unsure why, but when this is removed I get an error: "Caused by: ca.uhn.hl7v2.HL7Exception: 'CE' in record 1 is invalid for version 2.7 at OBX-2(0)" which is true but I'm not sure why it's targeting v2.7?

Reply:
We actually don't need this and added a new change that we do need with a comment explaining why we need it.